src/coreclr/gc/gcconfig.h

```diff
@@ -105,6 +105,7 @@ class GCConfigStringHolder
     INT_CONFIG   (GCTotalPhysicalMemory,     "GCTotalPhysicalMemory",     NULL,                                0,                  "Specifies what the GC should consider to be total physical memory")                      \
     INT_CONFIG   (GCRegionRange,             "GCRegionRange",             NULL,                                0,                  "Specifies the range for the GC heap")                                                    \
     INT_CONFIG   (GCRegionSize,              "GCRegionSize",              NULL,                                4194304,            "Specifies the size for a basic GC region")                                               \
+    INT_CONFIG   (GCEnableSpecialRegions,    "GCEnableSpecialRegions",    NULL,                                0,                  "Specifies to enable special handling some regions like SIP")                             \
```

Comment:
do we want to document this, or is it only for certain workloads we know would benefit from SIP?

Reply:
for now it's just special workloads so I thought I'd leave it as an experimental thing (so not supported in runtimeconfig). we can decide whether to support in runtimeconmfig in the next release when we have experimented with it more. does that sound reasonable?